ontology/mop.md

```diff
@@ -23,4 +23,5 @@ products:
 activity_status: active
 repository: https://github.com/rsc-ontologies/rxno
 preferredPrefix: MOP
+domain: biochemistry
```

Comment:
@batchelorc thoughts?

Reply:
also chemistry

Reply:
Unfortunately they did not make a chemistry specific domain, and you can only pick one, so this is the closest. The options are:

- upper
- anatomy and development
- health
- phenotype
- investigations
- environment
- biochemistry
- microbiology
- agriculture
- diet, metabolomics, and nutrition
- biological systems
- information technology
- organisms
- information
- simulation

Reply:
@matentzn is there a good reason why we shouldn't add a straight `chemistry` domain? Seems pretty wild that it didn't make the cut

Reply:
I dont know the difference - we could rename biochemistry to `chemistry` or `chemistry and biochemistry`? Maybe adding `chemistry` and `biochemistry` makes it hard for some people to chose which is more suitable?

Reply:
I like the idea of adding the "and" for chemistry and biochemistry. Tbh I don't want to get in the politics of categorizing ontologies (e.g., I think the diet/nutrition group that somehow co-oped metabolomics rather than that being in chemistry is pretty odd) so maybe it's better to just generalize stuff when possible

Reply:
Feel free to make a ticket and PR and I will drive it through..

Reply:
```suggestion
domain: chemistry and biochemistry
```